cpp/src/arrow/compute/kernels/hash_aggregate_test.cc

```diff
@@ -185,7 +184,8 @@ Result<Datum> GroupByUsingExecPlan(const BatchesWithSchema& input,
 Result<Datum> GroupByUsingExecPlan(const std::vector<Datum>& arguments,
```

Comment:
Perhaps this could just be called `GroupBy` now.

Reply:
I'm a little hesitant to use `GroupBy` since that would mean the `GroupBy` function calls the `GroupBy` function which is a little confusing.  I changed it to `RunGroupBy`.